src/main/java/module-info.java

```diff
@@ -0,0 +1,11 @@
+
+/**
+ * @author Moritz Thoma
+ * @since 1.0.0
+ */
+
+module playgroundFx {
```

Comment:
Modulname am besten wie Package-Name (gilt für alle Klassen): in Package `de.dhbw.silencio.ui`